test/LinalgExt/tile_to_in_parallel.mlir

```diff
@@ -0,0 +1,51 @@
+// RUN: mlir-proto-opt %s -linalg-tile-to-in-parallel| FileCheck %s
+
+// CHECK-DAG: #[[$CEIL_MAP:.*]] = affine_map<(d0)[s0] -> (d0 ceildiv s0)>
+// CHECK-DAG: #[[$MUL_MAP:.*]] = affine_map<(d0)[s0] -> (d0 * s0)>
+// CHECK-DAG: #[[$SUB_MAP:.*]] = affine_map<(d0, d1) -> (d0 - d1)>
+// CHECK-DAG: #[[$ID1_MAP:.*]] = affine_map<(d0) -> (d0)>
+// CHECK-DAG: #[[$ID2_MAP:.*]] = affine_map<(d0, d1) -> (d0, d1)>
+
+// CHECK-LABEL: func @static_tile
+//  CHECK-SAME:   %[[CHUNK_SIZE:[0-9a-z]+]]: index
+//  CHECK-SAME:   %[[IN:[0-9a-z]+]]: tensor<?xf32>
+//  CHECK-SAME:   %[[OUT:[0-9a-z]+]]: tensor<?xf32>
```

Comment:
CHECK-SAME means that they should appear on the same line?

Reply:
yes